### ISSUE TITLE
Train_drnet get stuck indefinitely while getting training batch

### ISSUE THREAD

**Issue by prashkmr** [edenton/drnet-py#7]
The data processing for loading the dataset is getting stuck in the below module: It has been stuck for 41 hours and I had to kill the process.

def get_training_batch():
    while True:
        print('In get_training_batch()')
        
        for sequence in train_loader:
            print("normalize_data")
            batch = utils.normalize_data(opt, dtype, sequence)
            yield batch

I have a 1080Ti GPU running and have even reduced the batch size and size of images to 64*64.
Nit able to understand what is causing the issue.

Any help is highly appreciated.

Thanks,
Prashant

**Comment by prashkmr** [comment]
Got it, actually, the num_workers should be set to 0 in some cases because the PyTorch parallel threads for loading a batch get hanged dur to some or the other reason.
In the code, the number of workers is set to a non zero number which can cause the issue.
